modules/utxo-bin/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2011-2017 bitcoinjs-lib contributors
```

Comment:
we should standardize the licenses for each package, let's handle in a follow up